modules/rostests/apitests/include/apitest.h

```diff
@@ -55,6 +55,7 @@
 #define ok_hr_(file, line, status, expected)    ok_hex_(file, line, status, expected)
 
 #define ok_eq_print(value, expected, spec)  ok((value) == (expected), #value " = " spec ", expected " spec "\n", value, expected)
+#define ok_eq_print_(file, line, value, expected, spec)  ok_(file,line)((value) == (expected), #value " = " spec ", expected " spec "\n", value, expected)
```

Comment:
```suggestion
#define ok_eq_print_(line, value, expected, spec) \
    ok((value) == (expected), \
       "Line %u: " #value " = " spec ", expected " spec "\n", line, value, expected)
```

Reply:
What's the point of that change?

Reply:
The new macro does not take a file path string argument, and if the test fails, it prints both line numbers that are passed into the function.